test/dlc_tests/conftest.py

```diff
@@ -643,6 +677,25 @@ def generate_unique_values_for_fixtures(metafunc_obj, images_to_parametrize, val
     return fixtures_parametrized
 
 
+def lookup_condition(lookup, image):
+    """
+    Return true if the ECR repo name ends with the lookup or lookup contains job type or device type part of the image uri.
+    """
+    #Extract ecr repo name from the image and check if it exactly matches the lookup (fixture name)
+    repo_name = image.split("/")[-1].split(":")[0]
+
+    job_type = ("training", "inference",)
+    device_type = ("cpu", "gpu", "eia", "neuron", "hpu")
+
+    if not repo_name.endswith(lookup):
```

Comment:
So just for my understanding, this currently makes the assumption that we have 3 different types of image parameterization fixtures:
1) the end of the repo name (why not just exact match on the repo name?)
2) exact match in one of the job types defined here, and job type present in image
3) exact match in one of the device types defined here, and device type present in image

Reply:
The conditions are correct. `endswith()` is used to replicate the existing logic.